boards/shields/m5stack_core2_ext/Kconfig.defconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+# SPDX-License-Identifier: Apache-2.0
+
+if M5STACK_CORE2_EXT
+
+config SENSOR
+	default y
```

Comment:
Board/shields should not enable subsystems/drivers, see also https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#general-recommendations

Reply:
So it's ok then, if the shield isn't working without enabling sensor support?
Should this dependency be highlighted in the documentation or what's the proper way to handle it?

Reply:
It is up to your application to enable/disable something. Shields are more about configuration, not enabling. A shield can have multiple configurations for e.g. display and sd card.